client/src/util/apollo.js

```diff
@@ -19,22 +19,35 @@ const WS_URI =
     ? "wss://brisk-hospitable-indianelephant.gigalixirapp.com/socket"
     : "ws://localhost:4000/socket";
 
+// Socket with full fallback to LongPoll
+// via https://elixirforum.com/t/fall-back-to-longpoll-when-websocket-fails/23894
+const socket = new PhoenixSocket(WS_URI, {
+  params: () => {
+    if (Cookies.get("token")) {
+      return { token: Cookies.get("token") };
+    } else {
+      return {};
+    }
+  },
+});
+socket.onError(() => {
+  if (navigator.onLine) {
+    console.error(
+      `Error connecting using ${
+        socket.transport == window.WebSocket ? "WebSocket" : "LongPoll"
+      }`,
+    );
+    if (socket.transport == window.WebSocket) socket.transport = LongPoll;
+    else if (window.WebSocket) socket.transport = window.WebSocket;
+  }
+});
+
 export const createClient = ({ ssr, req, fetch, tokenCookie } = {}) => {
   let link = createHttpLink({ uri: HTTP_URI, fetch });
   let absintheSocket;
 
   if (!ssr) {
-    absintheSocket = AbsintheSocket.create(
```

Comment:
Can you leave the `PhoenixSocket` creation inside this `if (!ssr) {` block? Otherwise SSR fails because the `PhoenixSocket` uses the browser global `WebSocket` which is not present on the server.

Reply:
Ah yes, sorry I missed that - refactored and repushed.